indico/modules/rb/__init__.py

```diff
@@ -40,7 +40,8 @@
     'vc_support_emails': [],
     'notification_hour': 6,
     'notification_before_days': 1,
-    'notifications_enabled': True
+    'notifications_enabled': True,
+    'booking_limit': 365
```

Comment:
I think it doesn't need to be changed. This is the global limit (days) for all the rooms and if a certain room has a custom booking limit, then this one will be used instead. Concerning `TimedeltaConverter` I can only see a couple of usages in our code base and these are in a test file, so I'm not sure how to use it.